packages/@uppy/informer/types/index.d.ts

```diff
@@ -1,26 +1,18 @@
-import Uppy = require('@uppy/core');
+import Uppy = require('@uppy/core')
 
 declare module Informer {
   interface Color {
-    bg: string | number;
-    text: string | number;
+    bg: string | number
+    text: string | number
   }
 
   interface InformerOptions extends Uppy.PluginOptions {
-    typeColors: {
-      [type: string]: Color
-    };
+    replaceTargetContent?: boolean
+    target?: Uppy.PluginTarget
+    typeColors?: { [type: string]: Color }
```

Comment:
I don't see this property used anywhere in the plugin 🤔

Reply:
yep, not anymore :D will remove